cli/core/pkg/auth/tkg/cluster_pinniped_info.go

```diff
@@ -19,25 +19,13 @@ import (
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 
 	netutil "github.com/vmware-tanzu/tanzu-framework/cli/core/pkg/auth/utils/net"
+	"github.com/vmware-tanzu/tanzu-framework/pinniped-components/common/pkg/pinnipedinfo"
```

Comment:
I'll be honest, with our `require/replace` and definition of a new module in `pinniped-components/common` I find the full path required in these other files odd.  I'm sure this will be resolved when the time comes to move this code to a separate repo.